cli/app.go

```diff
@@ -139,3 +157,37 @@ func runApps(args *docopt.Args, client *controller.Client) error {
 	}
 	return nil
 }
+
+func runInfo(_ *docopt.Args, client *controller.Client) error {
+	appName := mustApp()
+
+	fmt.Println("===", appName)
+
+	w := tabWriter()
+	defer w.Flush()
+
+	if remotes, err := gitRemotes(); err == nil {
```

Comment:
Every app can be technically deployed via git, currently there isn't a very tight coupling. The only thing we could do is not show the git repo if slugrunner is not in use for the current release (if there are no releases we should show it).